frontend/src/component/admin/billing/BillingDashboard/BillingPlan/BillingPlan.tsx

```diff
@@ -82,6 +82,7 @@ export const BillingPlan: FC<IBillingPlanProps> = ({ instanceStatus }) => {
         [InstancePlan.PRO]: 80,
         [InstancePlan.COMPANY]: 0,
         [InstancePlan.TEAM]: 0,
+        [InstancePlan.ENTERPRISE]: 0,
```

Comment:
According to https://github.com/Unleash/unleash/pull/3997/files#r1232089348 I don't think we need this.
```suggestion
```